cmake/test.cmake

```diff
@@ -1,5 +1,15 @@
 set(MAYA_USD_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 
+if(MayaUsd_FOUND)
```

Comment:
So here this should be either `MAYAUSD_MOD_PATH` or if you must have MAYAUSD_MOD_PATH then add it to the `REQUIRED_VARS` in FindMayaUsd.cmake.